app/src/main/res/layout/listitem_bowgun_detail.xml

```diff
@@ -0,0 +1,192 @@
+                app:layout_constraintTop_toTopOf="@+id/ammo_header">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:layout_gravity="center"
+                    android:text="Shot Type"
+                    android:textColor="@color/textColorHigh" />
+            </FrameLayout>
+
+            <FrameLayout
+                android:id="@+id/capacity_icon"
+                android:layout_width="@dimen/width_break_value"
+                android:layout_height="match_parent"
+                app:layout_constraintBottom_toBottomOf="@id/ammo_header"
+                app:layout_constraintEnd_toStartOf="@+id/reload_icon"
+                app:layout_constraintTop_toTopOf="@+id/ammo_header">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:layout_gravity="center"
+                    android:text="CAP"
+                    android:textColor="@color/textColorHigh" />
+            </FrameLayout>
+
+            <FrameLayout
+                android:id="@+id/reload_icon"
+                android:layout_width="@dimen/width_break_value"
+                android:layout_height="match_parent"
+                app:layout_constraintBottom_toBottomOf="@id/ammo_header"
+                app:layout_constraintEnd_toStartOf="@+id/special_icon"
+                app:layout_constraintTop_toTopOf="@+id/ammo_header">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:layout_gravity="center"
+                    android:text="RLD"
+                    android:textColor="@color/textColorHigh" />
+            </FrameLayout>
+
+            <FrameLayout
+                android:id="@+id/special_icon"
+                android:layout_width="@dimen/width_extract_value"
+                android:layout_height="match_parent"
+                android:layout_marginEnd="14sp"
+                app:layout_constraintBottom_toBottomOf="@id/ammo_header"
+                app:layout_constraintEnd_toEndOf="@+id/ammo_header"
+                app:layout_constraintTop_toTopOf="@+id/ammo_header">
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:layout_gravity="center"
+                    android:text="SPC"
+                    android:textColor="@color/textColorHigh" />
+            </FrameLayout>
+
+        </android.support.constraint.ConstraintLayout>
+
+        <LinearLayout
+            android:id="@+id/ammo_layout"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical">
+
```

Comment:
Add a comment here stating that this layout will be populated dynamically through code.